### ISSUE TITLE
Download NodeJS fail

### ISSUE THREAD

**Issue by davthu** [node-gradle/gradle-node-plugin#8]
What could be wrong when It seems like the `distbaseUrl` is not setup correctly because it adds `org.nodejs/` to the path?

```
node {
  version = '10.15.0'
  distBaseUrl = 'https://nodejs.org/dist'
}
```
Results in Error:
```
> Could not resolve all dependencies for configuration 'detachedConfiguration1'.
   > Could not find org.nodejs:node:10.15.0.
     Searched in the following locations:
         https://nodejs.org/dist/org.nodejs/node/10.15.0/ivys/ivy-10.15.0.xml
         https://nodejs.org/dist/org.nodejs/node/10.15.0/tar.gzs/node-tar.gz-linux-x64-10.15.0.tar.gz
         https://nodejs.org/dist/org.nodejs/node/10.15.0/tar.gzs/ivy-10.15.0.xml
```

**Comment by deepy** [comment]
I've tried to reproduce this and I can't seem to get it to fail, what Gradle version are you using?
If there's an error in the code it's likely [here](https://github.com/node-gradle/gradle-node-plugin/blob/9368ad44c369a92a9829f179f485957f0a84d902/src/main/groovy/com/moowork/gradle/node/task/SetupTask.groovy#L181-L188).

I tried this with Gradle 5.1 (since I assumed the issue came in 5.0) and Gradle 4.10, both download just fine on my Windows laptop.

    > :nodeSetup > Resolve files of :detachedConfiguration1 > node-v10.15.0-win-x64.zip > 12.48 MB/15.50 MB downloaded
